docs/source/reference/config-user-env.md

```diff
@@ -141,31 +122,25 @@ How you configure user environments for each category can differ a bit
 depending on what Spawner you are using.
 
 The first category is a **shared system (multi-user host)** where
-each user has a JupyterHub account and a home directory as well as being
+each user has a JupyterHub account, a home directory as well as being
 a real system user. In this example, shared configuration and installation
 must be in a 'system-wide' location, such as `/etc/`, or `/usr/local`
 or a custom prefix such as `/opt/conda`.
 
 When JupyterHub uses **container-based** Spawners (e.g. KubeSpawner or
-DockerSpawner), the 'system-wide' environment is really the container image
-that you are using for users.
+DockerSpawner), the 'system-wide' environment is really the container image used for users.
 
 In both cases, you want to _avoid putting configuration in user home
-directories_ because users can change those configuration settings. Also,
-home directories typically persist once they are created, so they are
-difficult for admins to update later.
+directories_ because users can change those configuration settings. Also, home directories typically persist once they are created, thereby making it difficult for admins to update later.
 
 ## Named servers
 
-By default, in a JupyterHub deployment, each user has exactly one server.
+By default, in a JupyterHub deployment, each user has one server only.
 
 JupyterHub can, however, have multiple servers per user.
-This is most useful in deployments where users can configure the environment
-in which their server will start (e.g. resource requests on an HPC cluster),
-so that a given user can have multiple configurations running at the same time,
-without having to stop and restart their one server.
+This is mostly useful in deployments where users can configure the environment in which their server will start (e.g. resource requests on an HPC cluster), so that a given user can have multiple configurations running at the same time, without having to stop and restart their own server.
 
-To allow named servers:
+To allow named servers, use this:
```

Comment:
I'd rephrase to something like "include this code snippet in your config file". "use this" isn't very explanatory.